addon/components/sortable-group.js

```diff
@@ -24,16 +29,20 @@ export default Component.extend({
         });
     },
     init() {
+        const fastboot = get(this, 'fastboot');
         //Default sortable group array to be null, these will be added when the sortable groups insert into the DOM
-        const sortable = new Sortable([], {
-            draggable: '.sortable-item',
-            mirror: {
-                constrainDimensions: true
-            },
-            plugins: [Plugins.ResizeMirror]
-        });
-        set(this, 'sortable', sortable);
-        this.initializeEventListeners();
+        if (!fastboot || get(fastboot, 'isFastBoot') === false) {
```

Comment:
i think it should be safe to simplify this to `if(!get(this, 'fastboot.isFastBoot')) { ...`, since even if the fastboot service does not exist, this'll return `undefined`